src/main/java/org/openelisglobal/common/controller/BaseMenuController.java

```diff
@@ -139,9 +137,9 @@ protected List<T> doNone(AdminOptionMenuForm<T> form, HttpServletRequest request
 
         if (getPageSize() > 0 && samePageList.size() > getPageSize()) {
             request.setAttribute(NEXT_DISABLED, "false");
-            // chop off last record (this was only to indicate that there are
-            // more records
-            samePageList = samePageList.subList(0, getPageSize());
+            // chop off last record (this was only to indicate that there are more records
+            //TODO: sets samePageList to 20 records only, trynna figure out why?
+//            samePageList = samePageList.subList(0, getPageSize());
```

Comment:
@mozzy11 i have temporarily commented out this line because it was limiting the number records sent by the /rest/dictionary-menu endpoint to 20, we have about 700 records. Not sure how to go about it but for now i needed the endpoint to be able to show me all those.

Reply:
got it, thanks!

will push a commit to do the needful